### ISSUE TITLE
Update to 2018 edition

### ISSUE THREAD

**Issue by kosticmarin** [georust/geohash#32]
Could it be possible to update the crate to 2018 edition before 0.9.0 release?

`cargo fix --edition`
should do most of the work, only problems are the bench tests that use nightly and the
command fails with the output:

```
error[E0554]: #![feature] may not be used on the stable release channel
 --> benches/base.rs:1:1
  |
1 | #![feature(test)]
  | ^^^^^^^^^^^^^^^^^
```

But if its run on the nightly channel it seems to work, and the code compiles on the stable as well :D

`rustup run nightly cargo fix --edition`

**Comment by frewsxcv** [comment]
0.9.0 was published a few hours ago, but if you'd like to open a pull request migrating this crate to the 2018 edition, we can always publish a 0.9.1 release

**Comment by sunng87** [comment]
Fixed in #33
